cluster-provision/okd/scripts/provision.sh

```diff
@@ -208,10 +208,14 @@ oc adm policy add-cluster-role-to-user cluster-admin admin
 
 if [ "${CNAO}" == "true" ]; then
     # Apply network addons
-    oc create -f /manifests/cna/namespace.yaml
-    oc create -f /manifests/cna/network-addons-config.crd.yaml
-    oc create -f /manifests/cna/operator.yaml
-    oc create -f /manifests/cna/network-addons-config-example.cr.yaml
+
+    # TODO: pass CNAO version at gocli
+    cnao_version=0.25.0
+    cnao_path=/manifests/cnao/$cnao_version/
+    oc create -f $cnao_path/namespace.yaml
+    oc create -f $cnao_path/network-addons-config.crd.yaml
+    oc create -f $cnao_path/operator.yaml
+    oc create -f $cnao_path/network-addons-config-example.cr.yaml
```

Comment:
This should be created by the user, not during provision. That way users can decide what to deploy. That may be useful e.g. in case of kubemacpool where we want to deploy everything but kubemacpool while testing master.

Reply:
You mean the CR right ?

Reply:
Yes.

Actually, we need to create the example here, wait for all images to appear and then remove the CR and wait for the cleanup to be done. That way we cache all images but don't leave anything running.

Reply:
But CNAO does not support delete, I think.

Reply:
You delete the whole NetworkAddonsConfig CR, thanks to owner references, that removes all the deployed components. We don't support partial removal.

Reply:
ack, let's add this.